[2024-09-05T01:19:24.189+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-05T01:19:24.216+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: otro_dag.bash_task scheduled__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-05T01:19:24.224+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: otro_dag.bash_task scheduled__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-05T01:19:24.225+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-05T01:19:24.244+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_task> on 2024-09-04 00:00:00+00:00
[2024-09-05T01:19:24.251+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=101) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T01:19:24.252+0000] {standard_task_runner.py:63} INFO - Started process 104 to run task
[2024-09-05T01:19:24.253+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'otro_dag', 'bash_task', 'scheduled__2024-09-04T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_to_fail.py', '--cfg-path', '/tmp/tmppup3xqw4']
[2024-09-05T01:19:24.254+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask bash_task
[2024-09-05T01:19:24.313+0000] {task_command.py:426} INFO - Running <TaskInstance: otro_dag.bash_task scheduled__2024-09-04T00:00:00+00:00 [running]> on host f5e4db2ad780
[2024-09-05T01:19:24.437+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='otro_dag' AIRFLOW_CTX_TASK_ID='bash_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-04T00:00:00+00:00'
[2024-09-05T01:19:24.437+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-05T01:19:24.495+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-05T01:19:24.496+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here is the message: Hello,world! {data_compose}"']
[2024-09-05T01:19:24.506+0000] {subprocess.py:86} INFO - Output:
[2024-09-05T01:19:24.509+0000] {subprocess.py:93} INFO - Here is the message: Hello,world! {data_compose}
[2024-09-05T01:19:24.510+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-05T01:19:24.511+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-05T01:19:24.544+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=otro_dag, task_id=bash_task, run_id=scheduled__2024-09-04T00:00:00+00:00, execution_date=20240904T000000, start_date=20240905T011924, end_date=20240905T011924
[2024-09-05T01:19:24.593+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-05T01:19:24.609+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-05T01:19:24.617+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
